_extensions/latex-environment/latex-environment.lua

```diff
@@ -70,6 +70,19 @@ local function writeEnvironments(divEl)
         local beginEnv = '\\begin' .. '{' .. v .. '}'
         local endEnv = '\n\\end{' .. v .. '}'
         
+        -- check if custom options or arguments are present
+        -- and add them to the environment accordingly
+        local opts = divEl.attr.attributes['options']
+        local args = divEl.attr.attributes['arguments']
+        
```

Comment:
I wonder if this would be simpler as:

```
local beginEnv = '\\begin' .. '{' .. v .. '}'

local opts = divEl.attr.attributes['options']
if opts then
  beginEnv = beginEnv .. '[' .. opts .. ']'
end

local args = divEl.attr.attributes['arguments']
if args then
  beginEnv = beginEnv .. '{' .. args .. '}'
end
```

Would you be ok with this change (happy to make it branch / pr if you agree)?